### ISSUE TITLE
Suggestions to improve apply theme

### ISSUE THREAD

**Issue by gossi** [thomas-lowry/themer#3]
Hello 👋 

Happily using your plugin and have some suggestions to improve it regarding the apply theme functionality. That mostly happened because I run the plugin, applied a theme and nothing happened. 

- Apply to document (not only the current page, which is what I'd need)
- Remember the choice where the theme change should happen (I for my given use case wouldn't use selection at all - though it is always the default)
- Only make the selection choosable if there *IS* a selection

I wouldn't mind sending a PR for that. Looking at the repo it looked more like the dist code rather than the dev code.

**Comment by thomas-lowry** [comment]
Hey @gossi, I am working on all of this for the next release! Stay tuned and thanks for your suggestions.

**Comment by eholder24** [comment]
I've also run into the issue of running the plugin to apply a theme and nothing happening – @gossi how did you get around it/fix it?

**Comment by thomas-lowry** [comment]
@eholder24 Are the styles published?
I recorded a video that walks through the setup: https://youtu.be/aM8SkksBy3Y

**Comment by eholder24** [comment]
Yes, they were published...but I ended up deleting the themes and starting over with two separate files and seems to be working now. Very likely I did something wrong when creating the two themes from the same doc with nested style names
